src/engine/Input/Pointer.ts

```diff
@@ -236,7 +240,7 @@ export class Pointers extends Class {
          var x: number = e.pageX - Util.getPosition(this._engine.canvas).x;
          var y: number = e.pageY - Util.getPosition(this._engine.canvas).y;
          var transformedPoint = this._engine.screenToWorldCoordinates(new Vector(x, y));
-         var pe = new PointerEvent(transformedPoint.x, transformedPoint.y, 0, PointerType.Mouse, e.button, e);
+         var pe = new PointerEvent(transformedPoint.x, transformedPoint.y, e.pageX, e.pageY, 0, PointerType.Mouse, e.button, e);
```

Comment:
I will verify the screen coords match what we expect and maybe rename the current `lastScreenPos` to `lastPagePos`.